### DISCUSSION TITLE
setting folders to analyse

### DISCUSSION THREAD

**Discussion by akrea** [nextcloud/recognize#670] [Q&A] [answered]
Hi

I was wondering if there is a way to limit the folders, which should be analyzed. My problem is, that it analyses my whole music-folder which has 1k+ artists most of the have their face on an album cover, which gets analyzed - which I do not need or want. I can hardly find the people relevant to me in this haystack.

Any help would be appreciated!

**Comment by marcelklehr** [comment, marked as answer]
See https://github.com/nextcloud/recognize#ignoring-directories

**Reply by akrea** [reply]
Thank you!!

**Reply by mplu** [reply]
Sorry for digging an old subject, but when a folder contains a .media file, does it exclude the folder + subfolder or just the folder ?

**Reply by marcelklehr** [reply]
The filename is `.nomedia` and it will cause ignoring of the whole subtree
